sigma/plugins/fun/xkcd.py

```diff
@@ -2,24 +2,36 @@
 import aiohttp
 import discord
 
-
 async def xkcd(cmd, message, args):
 
-    comic_no = str(random.randint(1, 2171))
+    random_comic = True
```

Comment:
Can you check whether there are args and assign the `random_comic` flag directly; pseudocode:

```
random_comic = has flags
if random comic:
    do this
    return

do that
```

Reply:
Umm...

I don't really understand what you are asking here.

>Can you check whether there are args and assign the `random_comic` flag directly;

That's what I'm doing in Line 8 (checking whether there are args) and 14 (assigning the `random_comic` flag).

Reply:
do `random_comic =  args != None and not is_int(args[0])` to assign it directly instead of setting it to `True` or `False` based on conditionals.

Reply:
Fixed in fdfd8a7